scripts/files/fs.py

```diff
@@ -42,3 +49,55 @@ def exists(path: str) -> bool:
     if is_s3(path):
         return fs_s3.exists(path)
     return fs_local.exists(path)
+
+
+def _download_tiff_and_sidecar(target: str, file: str) -> str:
+    """
+    Download a tiff file and some of its sidecar files if they exist to the target dir.
+
+    Args:
+        target (str): target folder to write to
+        s3_file (str): source file
+
+    Returns:
+        downloaded file path
+    """
+    download_path = os.path.join(target, f"{ulid.ULID()}.tiff")
+    get_log().info("Download File", path=file, target_path=download_path)
+    write(download_path, read(file))
+    for ext in [".prj", ".tfw"]:
+        try:
+            write(f"{target.split('.')[0]}{ext}", read(f"{file.split('.')[0]}{ext}"))
+            get_log().info(
+                "Download tiff sidecars", path=f"{file.split('.')[0]}{ext}", target_path=f"{target.split('.')[0]}{ext}"
+            )
+        except:  # pylint: disable-msg=bare-except
+            pass
+    return download_path
+
+
+def download_tiffs_parallel_multithreaded(inputs: List[str], target: str, concurrency: int = 10) -> List[str]:
```

Comment:
`fs.py`  has been designed to provide generic functions to deal files within multiple file systems (currently local and s3). It could be good idea to allow multithreading while doing operations on multiple files. However, I would make it really generic (something like a `write_all()`) writting a list of files in a target as you do with `download_tiffs_parallel_multithreaded()`.
I think that the specificity of downloading tiffs and sidecar files should not be handled in here.

Reply:
I agree and I did struggle to figure out where to put this function, if you have a suggestion I would really appreciate it!
I would like to pull it out of standarding.py to make it more visisble because I think it could be resused and others might not find it if it is burried an unrelated file. 
Maybe it needs its own file in `scripts/files/`?

This is why I put it in `fs.py` but if we only care about the first option I am happy to move it to `fs_s3.py`
In `fs.py`:

1. read s3 - write local
2. read s3 - write s3
3. read local - write local (uncessary?)
4. read local - write s3 (probably not the best performance)

in `fs_s3.py`:

1. read s3 - write local (have to import fs_local.write which (to me) seems not right)
2. ~~read s3 - write s3 (to get this and above, have to import fs.write - circular import error)~~
3. ~~read local - write local (not possible but also uncessary?)~~
4. ~~read local - write s3 (not possible probably not the best performance)~~

Reply:
Having it in `fs.py` rather than `fs_s3.py` makes more sense as `fs.write` and `fs.read` already manage multiple file systems (local and s3) so you don't have to worry about them